### ISSUE TITLE
Reproducing the results on RCV1

### ISSUE THREAD

**Issue by soumyac1999** [Tencent/NeuralNLP-NeuralClassifier#56]
I've tried everything that was mentioned in reply to various issues but I am still not able to reproduce the results given in the README.
- I used the complete RCV1 dataset consisting of 23149 train and 781265 test documents
- Used GloVe 300d embeddings for initialization `glove.6B.300d.txt`
 
The results that I have got are:
-- | Micro F1 | Macro F1
-- | -- | --
Flat | 0.8354 | 0.5801
Hierarchical | 0.8360 | 0.5798

Can you share the json configuration files used for getting the values you have reported?

**Comment by ayushbits** [comment]
@liqunhit @coderbyr 
Even I am not able to reproduce the reported numbers in Table 3 using RCNN for both Hierarchical and Flat.
RCV1 dataset used is the original dataset from trec and glove pretrained embeddings of 300 dim.
Could you see what could be the issue ?

Thanks

**Comment by coderbyr** [comment]
maybe two reasons : 
1、actually we use 10% of the origin train as the dev set, and using the other as the train set, it may get different results
2、we did't try all groups of the hyper parameters,  there could be better results than what we got in the paper

**Comment by soumyac1999** [comment]
@coderbyr
1. I am pretty sure that the choice of the 10% dev set should not improve the results of the flat model from (0.7955, 0.5123) to (0.8354, 0.5801).
2. My concern is not getting better results; all I want to know is the set of hyperparameters which you have used to get the reported results on Flat RCV1 with TextCNN. I haven't changed the hyperparameters from what is given in conf.json. Can you please share the config file which you have used to get the reported results?
